docs/src/pages/components/snackbars/LongTextSnackbar.tsx

```diff
@@ -9,20 +9,9 @@ const action = (
   </Button>
 );
 
-const useStyles = makeStyles((theme: Theme) => ({
-  root: {
-    maxWidth: 600,
-    '& > * + *': {
-      marginTop: theme.spacing(2),
-    },
-  },
-}));
-
 export default function LongTextSnackbar() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Box sx={{ maxWidth: 600, '& > * + *': { mt: 2 } }}>
```

Comment:
This could be `:not(style)` instead of `*`?

```suggestion
    <Box sx={{ maxWidth: 600, '& > :not(style) + :not(style)': { mt: 2 } }}>
```

Reply:
Stack